### PR TITLE
[test/plugin] immigrate sofarpc scenario

### PR DESCRIPTION
**Pull request by arugal**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
#3583
___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: Jenkinsfile-Agent-Test-3 [hunk 1]
```diff
@@ -73,6 +74,11 @@ pipeline {
                                 sh 'bash test/plugin/run.sh --build_id=${BUILD_ID} canal-scenario'
                             }
                         }
+                        state('sofarpc 5.4.0-5.6.2 (23)') {
```
Comment (kezhenxu94) [inline Jenkinsfile-Agent-Test-3]: what a typo :-)

File: Jenkinsfile-Agent-Test-3 [hunk 2]
```diff
@@ -75,6 +76,11 @@ pipeline {
                                 sh 'bash test/plugin/run.sh --build_id=wl3_${BUILD_ID} canal-scenario'
                             }
                         }
+                        state('sofarpc 5.4.0-5.6.2 (23)') {
+                            steps {
+                                sh 'bash test/plugin/run.sh --build_id=${BUILD_ID} sofarpc-scenario'
```
Comment (wu-sheng) [inline Jenkinsfile-Agent-Test-3]: `--build_id=wl3_${BUILD_ID}` should replace --build_id=${BUILD_ID}

File: Jenkinsfile-Agent-Test-3 (hunk 1 above)
[the remark this answers is not in the document]
Comment (arugal) [reply Jenkinsfile-Agent-Test-3]: haha

File: Jenkinsfile-Agent-Test-3 (hunk 2 above)
Comment (arugal) [reply Jenkinsfile-Agent-Test-3]: done

**Comment by wu-sheng** [comment]
Group 1 seems having slots still.

**Pull request merged**